docs/make.jl

```diff
@@ -13,6 +13,7 @@ makedocs(
                   "rational.md",
                   "modn.md",
                   "modp.md",
+		            "transExt.md",
```

Comment:
Did you push? There still seems to be a big space here. Take a look at the output on GitHub.